test/e2e/case1_framework_deployment_test.go

```diff
@@ -115,7 +115,7 @@ var _ = Describe("Test framework deployment", Ordered, func() {
 			checkArgs(ctx, cluster, expectedArgs...)
 
 			By(logPrefix + "removing the framework deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case1ManagedClusterAddOnCR, "--timeout=90s")
```

Comment:
Yeah, fair enough :rofl: 

It seemed like the test had only been failing on the "latest" k8s? That didn't help me understand *why* it was happening, but it made me feel better about possibly blaming a change in the addon framework... Or do you think it's more likely that it just got lucky the one time on the older k8s version?

Reply:
I'm not sure, but I think we just got lucky on the older K8s. I'm not even sure updating the `addon-framework` would do much, but maybe it's gotten some updates that might help?

Reply:
Gave it a shot. Doesn't seem to be better, but it's an upgrade so maybe it's still relevant/worthwhile?
- https://github.com/open-cluster-management-io/governance-policy-addon-controller/pull/210#issuecomment-3229768813